x-pack/test/detection_engine_api_integration/security_and_spaces/tests/open_close_signals.ts

```diff
@@ -157,6 +161,79 @@ export default ({ getService }: FtrProviderContext) => {
           );
           expect(everySignalClosed).to.eql(true);
         });
+
+        it('should NOT be able to close signals with t1 analyst user', async () => {
```

Comment:
Looking good! Thanks for adding these functional tests -- I think this gives everyone what they need to test all the different permission permutations when building out new features. This'll go a long way @dhurley14! 🙂